tests/testsuite/features2.rs

```diff
@@ -2009,10 +2009,7 @@ fn minimal_download() {
         p.build_dir().rm_rf();
     };
 
-    // none
-    // Should be the same as `-Zfeatures=all`
-    p.cargo("check -Zfeatures=compare")
-        .masquerade_as_nightly_cargo()
```

Comment:
I don't think this should change. `compare` runs a special mode that compares the new feature resolver against the old.

Reply:
I saw [`-Z features`](https://github.com/rust-lang/cargo/blob/8529eb1c9641a75667d1fccd8025c3fe46c803d7/src/doc/src/reference/unstable.md#features) had been stabilized but missed it in [`core::features.rs`](https://github.com/rust-lang/cargo/blob/8529eb1c9641a75667d1fccd8025c3fe46c803d7/src/cargo/core/features.rs#L906). It should be fixed now